app/run-ble-test.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```

Comment:
I usually use Python or PowerShell for this type of stuff, so Bash is a bit of a foreign language to me... I'm not going to review this in detail unless you want me to try.